pypeit/reduce.py

```diff
@@ -652,20 +653,21 @@ def spec_flexure_correct(self, sobjs, basename):
             basename (str):
 
         """
-
         if self.par['flexure']['spec_method'] != 'skip':
             # Measure
             flex_list = flexure.spec_flexure_obj(sobjs, self.slits.slitord_id, self.reduce_bpm,
                                                  self.par['flexure']['spec_method'],
                                                  self.par['flexure']['spectrum'],
                                                  mxshft=self.par['flexure']['spec_maxshift'])
             # QA
-            flexure.spec_flexure_qa(sobjs, self.slits.slitord_id, self.reduce_bpm, basename, self.det, flex_list,
+            # TODO -- These needs to be turned back on!!
+            if False:
+                flexure.spec_flexure_qa(sobjs, self.slits.slitord_id, self.reduce_bpm, basename, self.det, flex_list,
```

Comment:
Can you just comment this code instead of have the `if False` statement?

Reply:
turned back on